packages/dictionary/test/fixtures/references/codeList_references/input.ts

```diff
@@ -35,14 +35,17 @@ const content: Dictionary = {
 					meta: {
 						default: 'Unknown',
 					},
-					restrictions: {},
+					restrictions: {
+						codeList: '#/SINGLE_VALUE',
+					},
 				},
 			],
 		},
 	],
 	references: {
 		ID_REG_EXP: '^[\\w\\s\\W]{5,}$',
 		SEX: ['Male', 'Female'],
+		SINGLE_VALUE: 'this',
```

Comment:
Used for a test ensuring that single values are wrapped in an array for codeList restrictions.